content/people/carina-antunes/contents.lr

```diff
@@ -0,0 +1,13 @@
+_template: people.html
+---
+name: Carina Antunes
+---
+institution: CERN
+---
+photo: carina-antunes.png
+---
+role: Invenio Developer
+---
+bio: leads Citadel Search
```

Comment:
Would you mind to add a capital L for `leads`, just to be harmonised with the other Bios.

Reply:
Fixed, sorry about that. Copy/Paste from MM :disappointed: .